resources/lib/webscraper.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0 (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""Implements static functions for scraping the VRT NU website(https://www.vrt.be/vrtnu/)"""
+
+from __future__ import absolute_import, division, unicode_literals
+
+try:  # Python 3
+    from urllib.error import HTTPError
+    from urllib.parse import unquote
+    from urllib.request import build_opener, install_opener, urlopen, ProxyHandler
+except ImportError:  # Python 2
+    from urllib2 import build_opener, HTTPError, install_opener, ProxyHandler, unquote, urlopen
+
+from bs4 import BeautifulSoup, SoupStrainer
```

Comment:
I wonder if we can move this to the part where it is used. This is just used in 2 specific cases: `get_categories()` en `get_video_attributes()`. Just to avoid unneeded imports.

Reply:
Done.